repository/src/main/java/org/alfresco/service/cmr/repository/ContentService.java

```diff
@@ -170,8 +170,18 @@ public ContentWriter getWriter(NodeRef nodeRef, QName propertyQName, boolean upd
      *
      * @return {@code true} if direct access URLs retrieving is supported for the node, {@code false} otherwise
      */
+    @Deprecated
     boolean isContentDirectUrlEnabled(NodeRef nodeRef);
```

Comment:
As mentioned above. Should be changed to a default method, calling the new implementation, something like this:
```suggestion
    @Deprecated
    default boolean isContentDirectUrlEnabled(NodeRef nodeRef)
    {
        return isContentDirectUrlEnabled(nodeRef, ContentModel.PROP_CONTENT);
    }
```